app/src/main/java/com/example/awesomelanguagelearning/home/ui/views/home/HomeListTitle.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.awesomelanguagelearning.home.ui.views.home
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.example.awesomelanguagelearning.core.ui.theme.AppTheme
+import com.example.awesomelanguagelearning.core.ui.views.TextTitle
+
+@Composable
+fun HomeListTitle(
+    firstText: String,
+    secondText: String,
```

Comment:
@igel911 maybe let's add more context-specific names to these parameters (not mandatory, just a tip)